src/Microsoft.TestPlatform.TestHostProvider/Hosting/AppxManifestFile.cs

```diff
@@ -16,17 +14,17 @@ internal static class AppxManifestFile
     /// AppxManifest filePath
     /// </param>
     /// <returns>ExecutableName</returns>
-    public static string GetApplicationExecutableName(string filePath)
+    public static string? GetApplicationExecutableName(string filePath)
     {
         if (File.Exists(filePath))
         {
             var doc = XDocument.Load(filePath);
-            var ns = doc.Root.Name.Namespace;
+            var ns = doc.Root!.Name.Namespace;
 
-            return doc.Element(ns + "Package").
-                Element(ns + "Applications").
-                Element(ns + "Application").
-                Attribute("Executable").Value;
+            return doc.Element(ns + "Package")!.
```

Comment:
Are we completely sure the xml is well structured and none of those elements are missing ?

Reply:
Other reply gives more details but short answer is no, I am not sure but I am preserving the current in-place behavior. I have already faced a bug where changing nullability cause a bug because we were relying on a NRE caught somewhere else as part of the logic.